charts/clusternet-agent/README.md

```diff
@@ -13,12 +13,27 @@ helm install clusternet-agent -n clusternet-system --create-namespace \
 Please update `PLEASE-CHANGE-ME` to your valid configurations, such as,
 
 ```console
-helm install clusternet-agent -n clusternet-system --create-namespace \
+$ helm install clusternet-agent -n clusternet-system --create-namespace \
   --set parentURL=https://192.168.10.10:6443 \
   --set registrationToken=07401b.f395accd246ae52d \
   clusternet/clusternet-agent
 ```
 
+**Note:** If bootstrapping authentication is **not supported** by the kube-apiserver in parent cluster (like k3s) , i.e. `--enable-bootstrap-token-auth=false` (which defaults to be false), please use serviceaccount token instead.
+
+```console
+# Get serviceaccount token from parent cluster.
+$ kubectl get secret -n clusternet-system -o=jsonpath='{.items[?(@.metadata.annotations.kubernetes\.io/service-account\.name=="cluster-bootstrap-use")].data.token}' | base64 --decode; echo
```

Comment:
Well, this command is not run in child cluster. Better not add this here, which may mislead users.

Reply:
Could we add a link on how to get or generate this token?